tests/integration/tables/file.cpp

```diff
@@ -24,12 +33,51 @@ namespace {
 const std::vector<std::string> kFileNameList{
     // In order to test MBCS support, here's a japanese word
     // that *should* mean "dictionary"
```

Comment:
I can confirm it _does_ mean dictionary, if we want to remove uncertainty from our wording here